packages/lesswrong/server/manualMigrations/2023-01-10-revertSubforumNotifSettings.ts

```diff
@@ -0,0 +1,104 @@
+/* eslint-disable no-console */
+import { LWEvents } from '../../lib/collections/lwevents';
+import { Subscriptions } from '../../lib/collections/subscriptions';
+import UserTagRels from '../../lib/collections/userTagRels/collection';
+import { getSqlClientOrThrow } from '../../lib/sql/sqlClient';
+import { getSchema } from '../../lib/utils/getSchema';
+import { registerMigration } from './migrationUtils';
+
+registerMigration({
+  name: "revertSubforumNotifSettings",
+  dateWritten: "2023-01-10",
+  idempotent: true,
+  action: async () => {
+    const db = getSqlClientOrThrow();
+    if (Subscriptions.isPostgres()) { // at time of writing, this is true
```

Comment:
Small thing: I prefer returning early rather than forcing the rest of the function to be indented an extra time